src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,24 @@
+package racingcar.domain
+
+import java.util.Random
+
+class Car(
+    var position: Int = INIT_POSITION,
+    private val movable: () -> Boolean,
+) {
+    fun move(): Int {
+        if (movable.invoke()) {
```

Comment:
invoke 대신 그냥 실행하는건 어떨까요?

```suggestion
        if (movable()) {
```

Reply:
앗 바로 실행이 되는군요 😅